### PR TITLE
Updating the PoCRNet (2606) and Kerleano (1804) networks

### PR DESCRIPTION
**Pull request by guenoledc**
We are close to go live.
If you could merge soon that would be great

### CODE REVIEW

File: _data/chains/eip155-1804.json [hunk 1]
```diff
@@ -17,5 +19,11 @@
   "shortName": "kerleano",
   "chainId": 1804,
   "networkId": 1804,
-  "explorers": []
+  "explorers": [
+    {
+      "name": "Lite Explorer",
+      "url": "https://ethereum-pocr.github.io/explorer/kerleano",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-1804.json:26]:
```suggestion
      "standard": "none"
```

File: _data/chains/eip155-2606.json [hunk 2]
```diff
@@ -2,17 +2,26 @@
...
+  "infoURL": "https://github.com/ethereum-pocr/pocrnet",
   "shortName": "pocrnet",
   "chainId": 2606,
   "networkId": 2606,
-  "explorers": []
+  "explorers": [
+    {
+      "name": "Lite Explorer",
+      "url": "https://ethereum-pocr.github.io/explorer/pocrnet",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-2606.json:24]:
```suggestion
      "standard": "none"
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by ligi** [comment]
your explorers are not EIP3091 compatible - e.g. see address

**Comment by guenoledc** [comment]
> 


> your explorers are not EIP3091 compatible - e.g. see address

Actually they are compatible with the https://eips.ethereum.org/EIPS/eip-3091

For instance https://ethereum-pocr.github.io/explorer/pocrnet/block/17299 or https://ethereum-pocr.github.io/explorer/kerleano/block/1058695

**Comment by ligi** [comment]
both URLs you mention 404
image

**Comment by guenoledc** [comment]
Except that there is a bug in my github pages. Will fix asap and let you know. But the explorer is EIP3091 compatible

**Comment by ligi** [comment]
I cannot validate that - to me it currently looks it is not compatible when it comes to addresses

**Comment by guenoledc** [comment]
> I cannot validate that - to me it currently looks it is not compatible when it comes to addresses

It's working now if you can check

But I don't understand the comment "when it comes to addresses"

**Review by ligi — APPROVED** [review]

**Comment by guenoledc** [comment]
thanks

**Pull request merged**